package.json

```diff
@@ -17,7 +17,8 @@
     "dotenv": "^11.0.0",
     "express": "^4.17.2",
     "express-validator": "^6.14.0",
-    "mongoose": "^6.1.6"
+    "mongoose": "^6.1.6",
+    "styled-components": "^5.3.3"
```

Comment:
```suggestion
```
Oops, looks like you added `styled-components` to the backend package.json again.